packages/framework/fluid-static/package.json

```diff
@@ -74,6 +74,10 @@
   },
   "typeValidation": {
     "version": "0.59.2000",
-    "broken": {}
+    "broken": {
+      "ClassDeclaration_FluidContainer": {
+        "forwardCompat": false
```

Comment:
@heliocliu, I believe this situation is similar to the one in my last PR [here](https://github.com/microsoft/FluidFramework/pull/9439/files#r840771705). Thoughts on if this is okay to stay in main, or if this PR should be in next?

Reply:
this should be fine in main (since i don't think there are any runtime object compatibility considerations here)